Styles/general.css

```diff
@@ -1,32 +1,49 @@
-@import url('https://fonts.googleapis.com/css2?family=Comfortaa:wght@400;700&family=Lobster+Two&family=Poiret+One&display=swap');
-@import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Righteous&display=swap');
-@import url('https://fonts.googleapis.com/css2?family=Montserrat+Alternates:wght@300;400;500;600&family=Montserrat:wght@400;500;600&family=Poppins&family=Raleway:wght@600&display=swap');
+@import url("https://fonts.googleapis.com/css2?family=Comfortaa:wght@400;700&family=Lobster+Two&family=Poiret+One&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Crete+Round&family=Righteous&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Montserrat+Alternates:wght@300;400;500;600&family=Montserrat:wght@400;500;600&family=Poppins&family=Raleway:wght@600&display=swap");
 
 h1 {
-    font-size: 2rem;
-    font-weight: 700;
-    font-family: 'Montserrat Alternates', sans-serif;
+  font-size: 2rem;
+  font-weight: 800;
+  font-family: "Montserrat Alternates", sans-serif;
 }
 
 h2 {
-    font-size: 1.5rem;
-    font-weight: 500;
-    font-family: 'Montserrat', sans-serif;
+  font-size: 1.6rem;
+  font-weight: 650;
+  font-family: "Montserrat Alternates", sans-serif;
 }
 
 h3 {
-    font-size: 1rem;
-    font-weight: 500;
+  font-size: 1.4rem;
+  font-weight: 500;
+  font-family: "Montserrat Alternates", sans-serif;
+}
+h4 {
+  font-size: 1.1rem;
+  font-weight: 400;
+  font-family: "Montserrat", sans-serif;
+}
+
+h5 {
+  font-size: 0.9rem;
+  font-weight: 300;
+  font-family: "Montserrat", sans-serif;
 }
 
+h6 {
+  font-size: 12px;
```

Comment:
Here, you can use rem here for font-size as other font sizes are also declared with rem

Reply:
right, but for very small values, px is appreciated, that's why i used px for this particular font size.